go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift-online/ocm-cli
 
-go 1.18
+go 1.21
```

Comment:
is the go version update required here? I'm wondering if it can/should be handled in a separate MR.

Reply:
OCM SDK uses latest golang : https://github.com/openshift-online/ocm-sdk-go/commit/1252182841bbe28fc65153d4c502d95a67b158cf#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6
We can create a separate MR here though.